### DISCUSSION TITLE
bhyve_next_watering is giving me log errors

### DISCUSSION THREAD

**Discussion by limestar** [sebr/bhyve-home-assistant#90] [Q&A]
I believe I don't have this part setup correctly.

I have this code in my automations.yaml, is this correct?

`- alias: BHyve next watering & rain delay finishing updater
  trigger:
  - platform: state
    entity_id: switch.zone_1_zone, switch.sprinkler_rain_delay
  - platform: homeassistant
    event: start
  action:
  - service: python_script.bhyve_next_watering
    data:
      entity_id: switch.zone_1_zone`

log error:
Logger: homeassistant.components.automation.bhyve_next_watering_rain_delay_finishing_updater
Source: components/automation/__init__.py:520
Integration: Automation (documentation, issues)
First occurred: November 13, 2021, 11:59:41 PM (1 occurrences)
Last logged: November 13, 2021, 11:59:41 PM

Error while executing automation automation.bhyve_next_watering_rain_delay_finishing_updater: Unable to find service python_script.bhyve_next_watering

**Comment by sebr** [comment]
Did you manually install the script from this repo? You might've missed this small note in the [README](https://github.com/sebr/bhyve-home-assistant#python-script):

> Bundled in this repository is a python_script which calculates a device's next watering time and when a rain delay is scheduled to finish.
> 
> Note: HACS does not install the script automatically and they must be added manually to your HA instance.

**Reply by limestar** [reply]
I attempted it several times, but I believe I'm doing it incorrectly.

I've uploaded bhyve_next_watering.py script under config/.bhyve

Then in my automations.yaml file:

- alias: BHyve next watering & rain delay finishing updater
  trigger:
  - platform: state
    entity_id: switch.zone_1_zone, switch.sprinkler_rain_delay
  - platform: homeassistant
    event: start
  action:
  - service: python_script.bhyve_next_watering
    data:
      entity_id: switch.zone_1_zone

**I've also tried adding it to my scripts.yaml file too, but basically kicks out the same error.

**Reply by sebr** [reply]
You need to add it to the `python_scripts` directory

https://www.home-assistant.io/integrations/python_script/

**Reply by limestar** [reply]
Ok I got this working, needed that python_script folder and needed to add python_script: to my config yaml. Restart and the automation with the script pulled up the new entity and sensor!
